component/MorseResultBox.js

```diff
@@ -18,6 +18,11 @@ const JoinerTypeToClass = {
   [JoinerTypes.single]: morse_styles.result_input_char_joiner,
 }
 
+const CursorTypeToClass = {
+  [CursorTypes.insert]: morse_styles.result_input_char_cursor_left,
```

Comment:
Myslím, že to bude čitelnější, když se budou styly jmenovat konzistentně s typem kurzoru. Tedy `result_input_char_cursor_insert` a `result_input_char_cursor_edit`.